### DISCUSSION TITLE
git push no longer re-triggering work flow

### DISCUSSION THREAD

**Discussion by flagrant99** [community/community#69813] [Actions] [answered]
### Select Topic Area

Question

### Body

I have an msBuild workflow

on:
  workflow_dispatch:
  push:
    branches: [ "releases/**" ]

When I detect a branch change I do

    & git commit -a -m $commentStr;
    & git push;

This used to re-trigger the same workflow but now it doesn't. 

How can I get a & git push to trigger the work flow?
I do not understand how/when this stopped working.

**Comment by tleerai** [comment]
Not sure if it's a problem with your code or not, but Github actions is having an outage of some kind right now I think.

**Comment by flagrant99** [comment, marked as answer]
steps:
    - uses: actions/checkout@v3
      with:
        **persist-credentials: false** # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token and we won't pick up self invoked push events
